backend/tournesol/tests/test_api_statistics.py

```diff
@@ -105,10 +105,15 @@ def test_video_stats(self):
         self.assertEqual(last_month_video_count, 2)
 
         comparison_count = video_poll["comparisons"]["total"]
-        last_month_comparison_count = video_poll["comparisons"]["added_last_month"]
+        #should this variable name be changed as well?
+        last_month_comparison_count = video_poll["comparisons"]["added_last_30_days"]
```

Comment:
Yes let's do it.

It could be renamed simply `added_last_30_days` (or `last_30days_comparison_count`)